examples/validation/components/user-form.js

```diff
@@ -60,6 +67,19 @@ class UserForm extends React.Component {
           />
         </Field>
 
+        <Field model="user.dob" errors={{ lessThan10 }} validateOn="change">
```

Comment:
Bah, missed this yesterday. Had it as a date field originally but realised that's more complex than I needed for an example! Probably not an issue though.